### DISCUSSION TITLE
小米汽车 SU7 马上就要发布了，来看看我们帮你米出的驾考答题

### DISCUSSION THREAD

**Discussion by xbn2002** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#75] [General]
- 8160816 **小米汽车SU7是小米集团旗下的首款电动轿跑车，于2023年11月正式公布，预计2024年上市。关于小米汽车SU7的保养，以下说法中，哪些是正确的？** 【多选题】
  - 8161816 小米汽车SU7的机油需要每5000公里更换一次
    - ❌ 不选。小米汽车 SU7 是一款纯电动汽车，没有机油，也不需要更换机油。
  - 8162816 小米汽车SU7的刹车油需要每2年或4万公里更换一次
    - ✔️ 选中。刹车油是汽车制动系统的重要组成部分，需要定期更换，以保证刹车效果和安全性。一般建议每 2 年或 4 万公里更换一次刹车油。
  - 8163816 小米汽车SU7的变速箱油需要每4年或8万公里更换一次
    - ✔️ 选中。变速箱油是汽车变速箱的润滑剂，需要定期更换，以保证变速箱的正常工作和寿命。一般建议每 4 年或 8 万公里更换一次变速箱油。
  - 8164816 小米汽车SU7的火花塞需要每3年或6万公里更换一次
    - ❌ 不选。火花塞是汽油车的点火装置，用于产生电火花，点燃混合气。小米汽车 SU7 是一款纯电动汽车，没有火花塞，也不需要更换火花塞。
  - 8165816 小米汽车SU7的空调滤芯需要每年或2万公里更换一次
    - ✔️ 选中。空调滤芯是汽车空调系统的过滤器，用于过滤进入车内的空气，提高空气质量和舒适度。一般建议每年或 2 万公里更换一次空调滤芯。
  - 8166816 小米汽车SU7搭载澎湃OS，解锁BL需要到小米社区申请，解锁后可能造成电机保险丝熔断
    - ✔️ 选中。自 2023 年 11 月 8 日起，如果您的设备 BL 锁处于上锁状态且设备搭载的系统为小米澎湃 OS，那么无论您的设备是否曾经解锁，当您希望获取 BL 锁解锁权限时您都需要通过「小米社区」申请。对于 Xiaomi SU7，BL 解锁后您的机车将无任何安全保障，易受恶意软件攻击，且有可能会造成电机保险丝熔断。

**Comment by s88868888** [comment]
6

**Comment by Littlechilde** [comment]
哈哈

**Comment by pierrecashon** [comment]
好题

**Comment by zhilinli1** [comment]
不还原，不够难（

**Comment by Diffused7245** [comment]
Deepseek回答（未训练）😁


小米汽车 SU7 是纯电动车（EV），其保养项目和传统燃油车有很大不同。根据电动车的特点，正确的保养说法应该是：

正确选项（多选题）
✅小米SU7的刹车油需要每2年或4万公里更换一次
（电动车仍需要刹车油，但更换周期可能比燃油车更长）

✅小米SU7的空调滤芯需要每年或2万公里更换一次
（空调滤芯是常规保养项目，和燃油车类似）

错误选项（不符合电动车特性）
❌小米SU7的机油需要每5000公里更换一次
（纯电动车没有内燃机，不需要换机油）

❌小米SU7的变速箱油需要每4年或8万公里更换一次
（电动车通常采用单速减速器，一般不需要更换变速箱油） 
【注：这个错了】

❌小米SU7的火花塞需要每3年或6万公里更换一次
（电动车没有火花塞，这是燃油车的保养项目）

❌小米SU7搭载澎湃OS，解锁BL需要到小米社区申请，解锁后可能造成电机保险丝熔断
（“解锁BL”通常指手机Bootloader，汽车不存在此操作；电机保险丝熔断与软件解锁无关，属误导选项）
【注：Deepseek看来不知道车机解锁，我记得大多数车机都是安卓系统，~~理论上能油门刹车一起踩进fastboot~~】
